workers/hello-workers/hello-worker.js

```diff
@@ -15,13 +15,13 @@ addEventListener('message', function (e) {
   }
 });
 
-let subWorker = new Worker("sub-worker.js");
-subWorker.addEventListener('message', function (e) {
-  console.log("Subworker has responeded to ping", e.data.type);
-});
-subWorker.postMessage({
-  type: "ping"
-});
+// let subWorker = new Worker("sub-worker.js");
```

Comment:
nope - i'll bring it back. did that because Chrome does not support child workers